lib/postgres.go

```diff
@@ -1289,7 +1289,7 @@ func (postgres *Postgres) UpsertBlockTx(tx *pg.Tx, blockNode *BlockNode) error {
 
 		TxMerkleRoot: blockNode.Header.TransactionMerkleRoot,
 		Version:      blockNode.Header.Version,
-		Timestamp:    blockNode.Header.TstampNanoSecs,
+		Timestamp:    uint64(blockNode.Header.TstampNanoSecs),
```

Comment:
This will be deprecated anyway, so I'm adding this change so the code complies.

Reply:
Yeah this is fine.